rules/ensure-image-scanning-enabled-cloud/raw.rego

```diff
@@ -0,0 +1,31 @@
+package armo_builtins
+import future.keywords.in
+
+# Check if image scanning enabled for EKS
+deny[msga] {
+	describe_repositories := input[_]
+	describe_repositories.apiVersion == "eks.amazonaws.com/v1"
+	describe_repositories.kind == "DescribeRepositories"
+	describe_repositories.metadata.provider == "eks"
+	repos := describe_repositories.data.Repositories
+	some repo in repos
+	not image_scanning_configured(repo)
+	
+
+	msga := {
+		"alertMessage": "image scanning is not enabled",
+		"alertScore": 3,
+		"packagename": "armo_builtins",
+		"failedPaths": [],
+		"fixPaths": [],
```

Comment:
Why aren't we configuring failed / fix paths here?

Reply:
There is a fix command. This is not a resource that users can access and change it's configurations (like k8s yamls). The remediation can only be done through aws cli (or aws website)